pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining.detailed-output

```diff
@@ -152,6 +146,15 @@ Message: Node build0-gstfj-ci-tests-worker-c-dcz9p is degraded
     nodes: build0-gstfj-ci-tests-worker-c-dcz9p
   Description: failed to drain node: build0-gstfj-ci-tests-worker-c-dcz9p after 1 hour. Please see machine-config-controller logs for more information
 
+Message: Cluster Version version is failing to reconcile (ClusterOperatorDegraded)
+  Since:       0s
+  Level:       Warning
+  Impact:      Update Stalled
+  Reference:   https://github.com/openshift/runbooks/blob/master/alerts/cluster-monitoring-operator/ClusterOperatorDegraded.md
```

Comment:
This would be a great location for the generic troubleshooting document. Degraded CO happens to be good for this case but not all CV Failing=True is degraded operators.

Reply:
Not all `Failing=True` is because of ClusterOperators, but since openshift/cluster-version-operator#746, `Failing=True` ClusterVersion will trigger a `ClusterOperatorDegraded` alert with `name="version"` labels.  And the runbook will point to this same location.  So I think this is fine here, but we should extend the runbook text with a section on `Failing=True`.

Reply:
Makes sense :+1: